scripts/nginx/rutorrent.sh

```diff
@@ -137,7 +146,7 @@ DSKSP
 @define('PHP_GZIP_LEVEL', 2, true);
 
 \$do_diagnostic = true;
-\$log_file = '/tmp/rutorrent_errors.log'; // path to log file (comment or leave blank to disable logging)
+\${LOG}_file = '/tmp/rutorrent_errors.log'; // path to log file (comment or leave blank to disable logging)
```

Comment:
Bad substitution here
```suggestion
\$log_file = '/tmp/rutorrent_errors.log'; // path to log file (comment or leave blank to disable logging)
```